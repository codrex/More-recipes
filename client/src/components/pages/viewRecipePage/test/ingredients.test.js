import React from 'react';
import renderer from 'react-test-renderer';
import Ingredients from '../ingredients/ingredients';

const props = {
  ingredients: ['ingredients 1', 'ingredients 2', 'ingredients 3'],
};

describe('Ingredients display component ', () => {
  test('expected to match empty snapshot', () => {
    const component = renderer.create(<Ingredients {...props} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    expect(tree).toBeInstanceOf(Object);
  });
});
