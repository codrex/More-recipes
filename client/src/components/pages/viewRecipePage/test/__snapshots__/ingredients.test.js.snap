// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ingredients display component  expected to match empty snapshot 1`] = `
<div
  className="col-12 "
  style={Object {}}
>
  <div
    className="lead list-item"
  >
    <h4
      className="list-item-text"
      data-index={0}
      onClick={undefined}
    >
      ingredients 1
    </h4>
  </div>
  <div
    className="lead list-item"
  >
    <h4
      className="list-item-text"
      data-index={2}
      onClick={undefined}
    >
      ingredients 2
    </h4>
  </div>
  <div
    className="lead list-item"
  >
    <h4
      className="list-item-text"
      data-index={3}
      onClick={undefined}
    >
      ingredients 3
    </h4>
  </div>
</div>
`;
